src/libs/ReportUtils.js

```diff
@@ -543,6 +543,20 @@ function hashLogin(login, range) {
     return Math.abs(hashCode(login.toLowerCase())) % range;
 }
 
+/**
+ * There is a possibility that if the Expense report has a negative total.
+ * This is because there are instances where you can get a credit back on your card,
+ * or you enter a negative expense to “offset” future expenses
+ *
+ * @param {Object} moneyRequestReport
+ * @returns {Number}
+ */
+function getMoneyRequestAmount(moneyRequestReport) {
```

Comment:
We already have [this method](https://github.com/Expensify/App/blob/6851675645dd37a08a9c34023ac7af3695631db1/src/libs/ReportUtils.js#L1537), can we just rename it and add to that?